### ISSUE TITLE
Multiline eslint-disable comments with leading asterisks fail to disable rules

### ISSUE THREAD

**Issue by BrainBacon** [eslint/eslint#9646]
* **ESLint Version:** 4.11.0
* **Node Version:** 9.1.0
* **npm Version:** 5.5.1

<details>
<summary>Configuration</summary>

```json
{
    "parserOptions": {
        "ecmaVersion": 5,
        "sourceType": "script",
        "ecmaFeatures": {}
    },
    "rules": {
        "eqeqeq": 2,
        "semi": 2
    },
    "env": {}
}
```

</details>

Currently the only way to put disable directives on separate lines is to omit any leading asterisks `*`.
```js
// THIS WORKS AS EXPECTED
/* eslint-disable
   eqeqeq,
   semi
*/
1 == true
/* eslint-enable */

//  DOES NOT WORK WITH LEADING ASTERISKS
/* eslint-disable
 * eqeqeq,
 * semi
 */
1 == true
/* eslint-enable */
```

It is a common pattern to separate multiple lines in comments with leading asterisks. My editor places these automatically when I press return. There is one consideration: making sure this style is compatible with any documentation parsers such as JSDoc, but this should be fine as long as those parsers use the double asterisk pattern `/**` at the start of the comment block.

[Reproduction Link](https://eslint.org/demo/#eyJ0ZXh0IjoiLyogZXNsaW50LWRpc2FibGVcbiAgIGVxZXFlcSxcbiAgIHNlbWlcbiovXG4xID09IHRydWVcbi8qIGVzbGludC1lbmFibGUgKi9cblxuLyogZXNsaW50LWRpc2FibGVcbiAqIGVxZXFlcSxcbiAqIHNlbWlcbiAqL1xuMSA9PSB0cnVlXG4vKiBlc2xpbnQtZW5hYmxlICovXG5cbi8qIGVzbGludC1kaXNhYmxlXG4gICBlcWVxZXFcbiAgIHNlbWlcbiAqL1xuMSA9PSB0cnVlIiwib3B0aW9ucyI6eyJwYXJzZXJPcHRpb25zIjp7ImVjbWFWZXJzaW9uIjo1LCJzb3VyY2VUeXBlIjoic2NyaXB0IiwiZWNtYUZlYXR1cmVzIjp7fX0sInJ1bGVzIjp7ImVxZXFlcSI6Miwic2VtaSI6Mn0sImVudiI6e319fQ==)

**Comment by not-an-aardvark** [comment]
Thanks for the report.

I'm not sure this is a bug -- as far as I know, we only support putting whitespace between `eslint-disable` and the rule name. We don't support putting any other characters there, including asterisks.

Sorry about the delay in replying to this issue.

**Comment by mysticatea** [comment]
I think this is an enhancement; we had not been intending to support the `*`s.
I will not object the proposal to support it.

**Comment by BrainBacon** [comment]
@not-an-aardvark No problem. Thanks for taking a look.
@mysticatea I agree that it's an enhancement. Is there anything I need to do to bring it up as an official proposal, or is the `enhancement` label enough?

**Comment by ilyavolodin** [comment]
If you want to change it into proposal, could you update first message to follow this template: https://eslint.org/docs/developer-guide/contributing/changes

**Comment by mysticatea** [comment]
Personally, I think that the example of this issue is enough understandable.
Next, it needs a champion and three :+1:s from @eslint/eslint-team to accept the enhancement.

**Comment by not-an-aardvark** [comment]
I think this would require TSC consensus since it's a core change.

**Comment by ilyavolodin** [comment]
*TSC Summary:* Currently ESLint directives don't support multi-line comments with leading asterisks.
*TSC Question:* Proposal is to allow this behavior.

**Comment by not-an-aardvark** [comment]
This was discussed at [last week's TSC meeting](https://github.com/eslint/tsc-meetings/blob/master/notes/2018/2018-02-15.md#support-multiline-eslint-disable-comments-with-leading-asterisks). The resolution was that the TSC doesn't think there is a big value-add from allowing  leading asterisks, so we are inclined to not accept this proposal. However, @mysticatea wasn't at the meeting, so we wanted to see if @mysticatea feels strongly that this should be accepted before we close the issue.

**Comment by not-an-aardvark** [comment]
Closing because the TSC generally wasn't in favor of this proposal due to the lack of a clear value-add.
